src/java.base/share/classes/java/lang/invoke/MethodHandle.java

```diff
@@ -268,7 +268,7 @@
  *
  * <h2>Usage examples</h2>
  * Here are some examples of usage:
- * <blockquote><pre>{@code
+ * {@snippet lang="java" :
 Object x, y; String s; int i;
```

Comment:
These lines can now be properly indented, and they should be.

Reply:
That isn't an unreasonable suggestion. However, I wanted to present this change in an easy-to-review fashion so limited the diffs to the bracketing tags.